Source/Cryptlex.Net/Factories/CryptlexClientFactory.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Cryptlex.Net.Core;
+using Cryptlex.Net.Core.Services;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Cryptlex.Net.Factories
+{
+    public class DefaultHttpClientFactory : IHttpClientFactory
+    {
+        public HttpClient CreateClient(string name)
+        {
+            var client = new HttpClient();
+            return client;
+        }
+    }
```

Comment:
Do not create your own HttpClients, that is not the way HttpClientFactory works
Use DefaultHttpClientFactory to call IHttpClientFactory and use the library's HttpMessageHandlers pool

Something like this:

```csharp
public class DefaultHttpClientFactory : IHttpClientFactory
    {
        private readonly IHttpClientFactory _httpClientFactory;
        public DefaultHttpClientFactory(IHttpClientFactory httpClientFactory)
        {
            _httpClientFactory = httpClientFactory;
      }
        public HttpClient CreateClient(string name)
        {
            var client = _httpClientFactory.Create(name);
            return client;
        }
    }
```